examples/server/server.cpp

```diff
@@ -3783,6 +3795,48 @@ int main(int argc, char ** argv) {
         return 0;
     });
 
+    if (state.load() == SERVER_STATE_ERROR) {
+        // HTTP Server could not bind the port
+        return 1;
+    }
+    
+    // load the model
+    if (!ctx_server.load_model(params)) {
+        state.store(SERVER_STATE_ERROR);
+        return 1;
+    } else {
+        ctx_server.init();
+        state.store(SERVER_STATE_READY);
+    }
+
+    LOG_INFO("model loaded", {});
+
+    const auto model_meta = ctx_server.model_meta();
```

Comment:
This line can now be removed, since we never read `model_meta` below this line.

Reply:
Doesn't need to be populated after the model loading for the /models API answer?

Reply:
This line is outside of the lambda function for `/models` endpoint (in other words, you already have it inside `const auto handle_models`)